eng/common/scripts/Submit-PullRequest.ps1

```diff
@@ -31,7 +31,7 @@ param(
   [string]$RepoName,
 
   [Parameter(Mandatory = $true)]
-  [string]$BaseBranch,
+  [string]$BaseBranch = "master",
```

Comment:
What prompted this change?

Reply:
It actually doesn't have to be, I removed it.